sambacc/schema/conf-v0.schema.json

```
@@ -256,6 +256,19 @@
           },
           "admin_password": {
             "type": "string"
+          },
+          "interfaces": {
+            "type": "object",
+            "properties": {
+              "include_pattern": {
+                "type": "string",
+                "description": "A regular expression that must match for a system interface\nto be included in the AD DC interfaces list.\n"
```

nit:

```suggestion
                "description": "A regular expression that must match for a network interface\nto be included in the AD DC interfaces list.\n"
```

and other occurrences.

---

Good point, I'll fix that in a follow up change, thanks!

---

https://github.com/samba-in-kubernetes/sambacc/pull/110